include/config/config_debug.h

```diff
@@ -26,7 +26,26 @@
  * This allows you to quickly test the level you're working on.
  * If you want the game to boot normally, just comment out the define again.
  */
-// #define TEST_LEVEL LEVEL_BOB
+// #define STARTUP_SPAWN_LEVEL LEVEL_BOB
```

Comment:
`TEST LEVEL` in the description should probably be renamed to match the new define names.

Though honestly I would debate `TEST_LEVEL` was a better define name simply because it's less likely to trick a user into thinking it's a proper global define like what the `START_LEVEL` define is for (though tbf this is in the debug config file). The other thing is simply just user familiarity with the `TEST_LEVEL` name since it's such a commonly used define, but this is all just my opinion after all.